Lib/fontTools/merge.py

```diff
@@ -858,6 +859,7 @@ def merge(self, fontfiles):
 			self._preMerge(font)
 
 		self.duplicateGlyphsPerFont = [{} for f in fonts]
+		self.fonts = fonts
```

Comment:
I'm really curious why you have to store a reference to `fonts` or `fontfiles` in the `Merger` instance. I don't see those attribute used anywhere else.. 😕